Services/CO.CDP.OrganisationInformation.Persistence/Migrations/20250925133856_WelshHealthAndSafetySectionForm.cs

```diff
@@ -0,0 +1,194 @@
+                    RETURNING id INTO q7Id_1_Id;
+
+                    INSERT INTO form_questions (guid, next_question_id, next_question_alternative_id, section_id, ""type"", is_required, title, description, ""options"",  caption, summary_title, ""name"", sort_order)
+                    VALUES('{Guid.NewGuid()}', NULL, NULL, sectionId, 1, true, 'WelshHealthAndSafetyQuestion_07_02_Title', NULL, " + healthAndSafety05MultiQuestionGroupedOptionsJson + $@", NULL, 'WelshHealthAndSafetyQuestion_07_02_SummaryTitle', '_WelshHealthAndSafetyQuestion07_2', 12)
+                    RETURNING id INTO q7Id_2_Id;
+
+                    INSERT INTO form_questions (guid, next_question_id, next_question_alternative_id, section_id, ""type"", is_required, title, description, ""options"",  caption, summary_title, ""name"", sort_order)
+                    VALUES('{Guid.NewGuid()}', NULL, NULL, sectionId, 10, true, 'WelshHealthAndSafetyQuestion_07_03_Title', NULL, " + healthAndSafety05MultiQuestionGroupedOptionsJson + $@", NULL, 'WelshHealthAndSafetyQuestion_07_03_SummaryTitle', '_WelshHealthAndSafetyQuestion07_3', 13)
+                    RETURNING id INTO q7Id_3_Id;
+
+                    INSERT INTO form_questions (guid, next_question_id, next_question_alternative_id, section_id, ""type"", is_required, title, description, ""options"",  caption, summary_title, ""name"", sort_order)
+                    VALUES('{Guid.NewGuid()}', NULL, NULL, sectionId, 2, false, 'WelshHealthAndSafetyQuestion_08_Title', 'WelshHealthAndSafetyQuestion_08_Description', " + healthAndSafety08OptionsWithInsetTextJson + $@", NULL, 'WelshHealthAndSafetyQuestion_08_SummaryTitle', '_WelshHealthAndSafetyQuestion08', 14)
+                    RETURNING id INTO q8Id;
+
+                    INSERT INTO form_questions (guid, next_question_id, next_question_alternative_id, section_id, ""type"", is_required, title, description, ""options"",  caption, summary_title, ""name"", sort_order)
+                    VALUES('{Guid.NewGuid()}', NULL, NULL, sectionId, 3, true, 'WelshHealthAndSafetyQuestion_09_Title', NULL, " + healthAndSafety06GroupedOptionsJson + $@", NULL, 'WelshHealthAndSafetyQuestion_09_SummaryTitle', '_WelshHealthAndSafetyQuestion09', 15)
+                    RETURNING id INTO q9Id;
+
+                    INSERT INTO form_questions (guid, next_question_id, next_question_alternative_id, section_id, ""type"", is_required, title, description, ""options"",  caption, summary_title, ""name"", sort_order)
+                    VALUES('{Guid.NewGuid()}', NULL, NULL, sectionId, 2, false, 'WelshHealthAndSafetyQuestion_10_Title', 'WelshHealthAndSafetyQuestion_10_Description', " + healthAndSafety07GroupedOptionsJson + $@", NULL, 'WelshHealthAndSafetyQuestion_10_SummaryTitle', '_WelshHealthAndSafetyQuestion10', 16)
```

Comment:
sort order isn't used by our application but might be relied on by eSenders. These final 3 questions have conflicting sort orders, there are two 16s and check your answers should be the highest